dash-renderer/src/actions/index.js

```diff
@@ -560,6 +568,8 @@ function updateOutput(
             })
         );
 
+        dispatch(setAppIsReady());
```

Comment:
In principle we should be able to do a lot less work than this - only walk the new props, and chain the resulting promise (if any) onto any existing isReady promise.